packages/admin-ui-components/babel.config.js

```diff
@@ -17,6 +17,7 @@ const presets = [
 const plugins = [
   "@babel/proposal-class-properties",
   "@babel/proposal-object-rest-spread",
+  ["import", { "libraryName": "antd", "style": "css" }]
```

Comment:
This configuration will require Antd components from `ant/lin/<component>` with relevant styles, instead of loading entire package and styles.